server/src/main/java/org/apache/iotdb/db/utils/QueryUtils.java

```diff
@@ -127,12 +128,27 @@ public static void constructPathAndDataTypes(List<PartialPath> paths, List<TSDat
     paths.add(new PartialPath(COLUMN_TIMESERIES_COMPRESSION, false));
     dataTypes.add(TSDataType.TEXT);
 
-    Set<String> tagAndAttributeName = new TreeSet<>();
+    //check if timeseries result has tag or attribute
+    boolean hasTag = false;
+    boolean hasAttribute = false;
     for (ShowTimeSeriesResult result : timeseriesList) {
```

Comment:
I don't think there's any need to check,  it's ok to show `null || spaces`, like :
```
+--------------+-----+-------------+--------+--------+-----------+----+----------+
|    timeseries|alias|storage group|dataType|encoding|compression|tags|attributes|
+--------------+-----+-------------+--------+--------+-----------+----+----------+
|root.sg1.d1.s1| null|     root.sg1|   INT64|     RLE|     SNAPPY|null|      null|
|root.sg1.d1.s2| null|     root.sg1|   INT64|     RLE|     SNAPPY|null|      null|
|root.sg1.d1.s3| null|     root.sg1|   INT64|     RLE|     SNAPPY|null|      null|
+--------------+-----+-------------+--------+--------+-----------+----+----------+

```